static/app/components/events/contexts/memoryInfo/getMemoryInfoKnownDataDetails.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  getMemoryInfoKnownDataDetails,
+  memoryInfoKnownDataValues,
+} from 'sentry/components/events/contexts/memoryInfo/getMemoryInfoKnownDataDetails';
+
+import {memoryInfoMockData} from './index.spec';
+
+describe('getMemoryInfoKnownDataDetails', function () {
+  it('returns values and according to the parameters', function () {
+    const allKnownData: ReturnType<typeof getMemoryInfoKnownDataDetails>[] = [];
+
+    for (const type of Object.keys(memoryInfoKnownDataValues)) {
+      const memoryInfoKnownData = getMemoryInfoKnownDataDetails({
+        type: memoryInfoKnownDataValues[type],
+        data: memoryInfoMockData,
+        event: TestStubs.Event(),
+      });
+
+      if (!memoryInfoKnownData) {
+        continue;
+      }
+
+      allKnownData.push(memoryInfoKnownData);
+    }
+
+    expect(allKnownData).toEqual([
+      {subject: 'Allocated Bytes', value: 9614872},
+      {subject: 'Fragmented Bytes', value: undefined},
```

Comment:
Since this is an "all known" test, should it be passing `undefined`?  I would expect items with `undefined` or `null` values to not appear in the UI.

Reply:
I have to double-check, maybe this happens in the function https://github.com/getsentry/sentry/blob/master/static/app/components/contextData/index.tsx,  but I am almost sure that we don't display empty values in the UI.